### PR TITLE
Create API endpoint for the statistics

### LINKED ISSUE #344: [client] Make use of the API to get buildings statistics
The statistic section on the main page is currently showing mock data. An endpoint will be created in the API returning the following statistics:
```JSON
{
    "people_under_risk": 1234,
    "evaluated_buildings": 567,
    "consolidated_buildings": 89
}
```
API issue: #337

### PR DESCRIPTION
**Pull request by geozaur**
### What does it fix?

[Make use of the API to get buildings statistics](https://github.com/code4romania/seismic-risc/issues/344)

Closes #344 

Retrieves statistics data from api on home page.

### How has it been tested?

I've tested it locally,

`STATISTICS_URL` constant might be changed.

### CODE REVIEW

File: client/src/containers/home/StatisticFragment.js [hunk 1]
```diff
@@ -10,17 +10,37 @@ const StatisticCard = (props) => (
...
 export default () => {
+  const [state, setState] = React.useState({
+    peopleUnderRisk: '',
+    evaluatedBuildings: '',
+    consolidatedBuilding: '',
+  });
+  React.useEffect(() => {
+    fetch(STATISTICS_URL)
+      .then((res) => res.json())
+      .then(({ people_under_risk, evaluated_buildings, consolidated_buildings }) => {
```
Comment (geozaur) [inline client/src/containers/home/StatisticFragment.js:24]: Snake case notation breaks eslint rules, if I go for the computed property notation (e.g: statistics['people_under_risk']) it still returns an error, suggesting dot notation.
I could disable the eslint rule for this line or we could change the overall eslint config 🤔

Comment (tudoramariei) [reply client/src/containers/home/StatisticFragment.js:24]: Can't we just change their names with a _camelCase_ version? If we're going to have conflicts if we have changes in `peopleUnderRisk: peopleUnderRisk`I think we can have it named as `peopleUnderRisk: peopleUnderRiskState` or something similar 🤔

Comment (geozaur) [reply client/src/containers/home/StatisticFragment.js:24]: There's no naming collisions, I just copied the names of the fields from the ticket in which the API was described - there were written in snake case, but if there's used a serializer or something that's great

Comment (tudoramariei) [reply client/src/containers/home/StatisticFragment.js]: Fair enough. Probably will be using camelcase for the endpoint 🤔 
Either this or I'll be refactoring it 😅

**Pull request merged**